plugins/source/aws/resources/services/kafka/helpers.go

```diff
@@ -32,6 +32,10 @@ func describeClustersInput(resource *schema.Resource) kafka.DescribeClusterV2Inp
 
 func resolveKafkaTags(path string) schema.ColumnResolver {
 	return func(ctx context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
+		if path == "ClusterOperations" {
```

Comment:
We're passing in `OperationArn` in the table definition, shouldn't that be used here as well?

Reply:
Yes